api/comments_api.py

```diff
@@ -48,17 +60,19 @@ def do_get(self, **kwargs) -> dict[str, list[dict[str, Any]]]:
     """Return a list of all review comments on the given feature."""
     feature_id = kwargs['feature_id']
     field_id = kwargs.get('field_id', None)
+    comments_only = self.get_bool_arg('comments_only')
     # Note: We assume that anyone may view approval comments.
     comments = Activity.get_activities(
-        feature_id, field_id, comments_only=True)
-    user = self.get_current_user(required=True)
+        feature_id, field_id, comments_only=comments_only)
+    user = self.get_current_user()
     is_admin = permissions.can_admin_site(user)
-    
+
     # Filter deleted comments the user can't see.
+    user_email = user.email() if user else None
     comments = list(filter(
-      lambda c: self._should_show_comment(c, user.email(), is_admin), comments))
+      lambda c: self._should_show_comment(c, user_email, is_admin), comments))
 
-    dicts = [comment_to_json_dict(c) for c in comments]
+    dicts = [activity_to_json_dict(c) for c in comments]
```

Comment:
maybe call it activities_json. dicts is too close to the type `dict`

Reply:
I am using the word `dict` here to indicate that it is a python `dict`.  It is a dict that will be used to generate json.  If the name just said `json`, that would seem to indicate a string in json format.